src/Microsoft.Data/PrimitiveDataFrameColumn.ReversedBinaryOperations.cs

```diff
@@ -0,0 +1,394 @@
+﻿
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Oops - the rebase left these in the old folder.

Reply:
Yup, I see it. Fixing it